src/aws_lambda.py

```diff
@@ -1,5 +1,62 @@
 import boto3
+import os
+
+from .s3 import S3
+
 
 class AWSLambda:
 
-    def __init__(self):
+    def __init__(self, project_name, bucket_name=None, function_name=None, runtime="python3.6"):
+        self.project_name = "{}-lambda".format(project_name)
+        self.bucket_name = bucket_name
+        self.function_name = function_name
+        self.runtime = runtime
+        self.client = boto3.client('lambda')
+        self.function_path = os.path.abspath(os.path.join(
+                                                     os.path.dirname( __file__ ),
+                                                     '..',
+                                                     'function.zip'
+                                                     )
+                                        )
+
+        if self.bucket_name:
+            self.s3 = S3(self.bucket_name)
+        else:
+            self.s3 = None
+
+    def deploy_function(self, model_bucket_name, model_name, role=None):
+        self.client.create_function(
+            FunctionName=self.project_name,
+            Runtime=self.runtime,
+            Role=self.role,
+            Handler='handler.predict',
```

Comment:
I think, at some point, we should create a Config class where things like this hard-coded string ('handler.predict')can be stored. Doesn't have to be a class per se, but some solution where defined strings/magic numbers/etc. are maintained so that they're all in one place.